applications/ExternalSolversApplication/external_includes/feast_solver.h

```diff
@@ -216,6 +220,25 @@ class FEASTSolver: public LinearSolver<TSparseSpaceType, TDenseSpaceType,
     {
     }
 
+    /**
+     * This method returns directly the first eigen value obtained
+     * @param rK: The stiffness matrix
+     * @param rM: The mass matrix
+     * @return The first eigenvalue
+     */
+    double GetEigenValue(
```

Comment:
Thats true, this is how existing eigen solvers are tested (also in the core) 
I agree that this is not a good option especially because feast, (and also the subspace iteration algorithm in the EigenSolversApplication) are calculating more than one eigen value.

Reply:
I think directly calling the Solve() function and interpreting their results in python would be the best option.

Reply:
Ok, I agree with that and then this function should not be necessary right?

Reply:
Right. It should probably be done in a separate PR because it would affect other solvers too?

Reply:
I don't remember if we discussed (in person) before or after this comment. But basically, removing GetEigenValue and relying on the Solve function in LinearSolver for the tests was the way to go.